test/test.js

```diff
@@ -3,21 +3,23 @@ var isRelative = require('../');
 
 describe('is-relative-path', function () {
   it('returns true if the given path is relative', function () {
+    assert.ok(isRelative(''));
```

Comment:
In node's documentation of path.isAbsolute():

> If  the given path is a zero-length string, false will be returned.

I want to verify the complement of that use case.